wagtail/wagtailcore/blocks/static_block.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import absolute_import, unicode_literals
+
+from django.utils.safestring import mark_safe
+
+from .base import Block
+
+__all__ = ['StaticBlock']
+
+
+class StaticBlock(Block):
+    """
+    A block that just 'exists' and has no fields.
+
+    It's an abstract model, so it has to be subclassed.
+    Subclasses should define self.admin_html in order to
+    display something in the admin even though the block
+    has no fields.
+    """
+    def render_form(self, value, prefix='', errors=None):
+        admin_html = getattr(self, 'admin_html', '')
+        return mark_safe(admin_html)
+
+    def value_from_datadict(self, data, files, prefix):
+        return None
+
+    class Meta:
+        abstract = True
```

Comment:
`Meta.abstract` does not have any meaning to blocks. The `abstract` attribute is only meaningful on models. This should be removed.